packages/near-membrane-base/src/membrane.ts

```diff
@@ -112,13 +112,19 @@ export type HooksCallback = (
 
 export type DistortionCallback = (target: ProxyTarget) => ProxyTarget;
 
+interface InstrumentationHooks {
```

Comment:
@garychangsf we need to figure what API to use here.

Reply:
O11y is versatile. We can stick to the activity API which has start() and stop(). We can cram extra data we want to log at the stop() call.

Reply:
We want to put this into a separate file (instrumentation.ts). Currently, Locker code base has defined a similar interface but I think we should have the same interface shared between Locker and Near-membrane.